.github/workflows/mkdocs-master.yml

```
@@ -0,0 +1,38 @@
+name: mkdocs
```

You might need to give them unique names here, so we can distinguish them in the Github UI where the runs are shown.
```suggestion
name: mkdocs-master
```

---

I set unique names on all PRs